Makefile

```diff
@@ -168,7 +168,7 @@ update-third-party:
 
 .PHONY: refresh-sample-testdata
 refresh-sample-testdata: out/$(SAMPLES_REPO)/.decompressed-$(SAMPLES_COMMIT) out/mal
-	./tests/refresh-testdata.sh ./out/mal out/$(SAMPLES_REPO)
+	./out/mal refresh
```

Comment:
I think you'll still need to take the directory as an argument, otherwise you won't process overrides correctly.